CONTRIBUTING.md

```diff
@@ -63,51 +68,51 @@ These tests are used to prevent visuals regressions in our components exported f
+- Add a four-space indented paragraph after each non-trivial list item, explaining what changed and why. For each breaking change also write who it affects and instructions for migrating existing code.
+- Maybe add some newlines here and there. Preview the result on GitHub to get a feel for it. Changelog generator output is a bit too terse for my taste, so try to make it visually pleasing and well grouped.
 
-### The steps to prepare and trigger a release to `next` are as follows:
+3. (_Optional_) Include "_Migrating from ..._" instructions for the previous release in case you deem it necessary.
+4. Commit the changelog (usually by opening a new Pull Request).
 
-- ensure you are on the latest `master` branch
-- update the `CHANGELOG.md`
-  - run `yarn changelog` to get a code snippet of the important commits from the last release
-  - copy that and paste it into `CHANGELOG.md` file
-  - make sure that the git tag references are correctly defined and the top entry represents the _new_ tag that you are about to create
-  - add or modify the generated changelog to provide more context about the new release
-- bump the `version` in the `package.json`
-- make a commit and push it to `master` (e.g. `git commit -am "chore: bump version to 2.0.0, update changelog"`)
-  - if you can't push it directly to `master`, open a Pull Request first
-- at this point you can create the `tag`: `git tag -m "Version v2.0.0" v2.0.0`
-  - the tag name is the `version` string in the `package.json` plus the prefix `v`
-- push the tag: `git push --tags`
+#### Release the packages
 
-From that point on, [CircleCI](https://circleci.com/gh/commercetools/ui-kit) will take over the release: build the bundles, publish to `npm` and update branch for the documentation website (see below).
+1. Make sure the `CHANGELOG.md` has been updated.
+2. Check that your npm account has access to the `@commercetools-frontend` and `@commercetools-uikit` organizations and that you are logged in with the `npm` CLI.
+3. Run `yarn release`: the packages will be bundled with Rollup first, then Lerna will prompt you to select the version that you would like to release (minor, major, pre-release, etc.).
+4. Wait a bit until Lerna bumps the versions, creates a commit and a tag and finally publishes the packages to npm (to the `next` distribution channel).
+5. After publishing, create a GitHub Release with the same text as the `CHANGELOG.md` entry. See previous Releases for inspiration.
 
-### Moving the `latest` dist-tag to a release:
+#### Moving the `latest` dist-tag to a release:
 
-After testing the `next` release on a production project, it's time to move to the `latest` dist-tag to make the release official.
+After testing the `next` release on a production project, if the version is **stable** it can be finally movede to the `latest` distribution channel.
```

Comment:
```suggestion
After testing the `next` release on a production project, if the version is **stable** it can be finally moved to the `latest` distribution channel.
```